Sources/_CryptoExtras/RSA/RSA_boring.swift

```diff
@@ -564,6 +591,73 @@ extension BoringSSLRSAPrivateKey {
             return output
         }
 
+        fileprivate func blindSignature<D: DataProtocol>(_ blindedMessage: D) throws -> _RSA.BlindSigning.BlindSignature {
+            let rsaPrivateKey = CCryptoBoringSSL_EVP_PKEY_get0_RSA(self.pointer)
+            let signatureByteCount = Int(CCryptoBoringSSL_RSA_size(rsaPrivateKey))
+
+            guard blindedMessage.count == signatureByteCount else {
+                throw CryptoKitError.incorrectParameterSize
+            }
+
+            let blindedMessageBytes: ContiguousBytes = blindedMessage.regions.count == 1 ? blindedMessage.regions.first! : Array(blindedMessage)
+
+            let signatureBytes = try Array<UInt8>(unsafeUninitializedCapacity: signatureByteCount) { signatureBufferPtr, signatureBufferCount in
+                try blindedMessageBytes.withUnsafeBytes { blindedMessageBufferPtr in
+                    /// NOTE: BoringSSL promotes the use of `RSA_sign_raw` over `RSA_private_encrypt`.
+                    var outputCount = 0
+                    guard CCryptoBoringSSL_RSA_sign_raw(
+                        rsaPrivateKey,
+                        &outputCount,
+                        signatureBufferPtr.baseAddress,
+                        signatureBufferPtr.count,
+                        blindedMessageBufferPtr.baseAddress,
+                        blindedMessageBufferPtr.count,
+                        RSA_NO_PADDING
+                    ) == 1 else {
+                        throw CryptoKitError.internalBoringSSLError()
+                    }
+                    precondition(outputCount == signatureByteCount)
```

Comment:
Did you test this on enough key sizes that it's always true? (eg. if first bytes are 0, they are not getting trimmed)

Just want to make sure we don't crash if we have an oddly formatted private key generated some day.

Reply:
Ah. Let me double check on this, thanks!